### PR TITLE
CRUD's

### PR DESCRIPTION
**Pull request by DarshanAIakhramsing**
Kijk plox

### CODE REVIEW

File: app/Http/Controllers/AnimeController.php [hunk 1]
```diff
@@ -38,51 +44,79 @@ public function create()
...
     {
-        //
+        return view('anime.show',compact('anime'));
     }
 
     /**
      * Show the form for editing the specified resource.
      *
-     * @param  \App\Anime  $anime
+     * @param  \App\Manga  $manga
```
Comment (Kokosnootje) [inline app/Http/Controllers/AnimeController.php]: Manga? This. Is. ANIME.

File: resources/views/anime/DoNotTouchPlox/animepage.blade.php [hunk 2]
```diff
@@ -56,7 +56,7 @@
         </th>
     </tr>
     <tr>
-            <td>koala</td>
+            <td><img src="{{asset('../public/images/UBW.jpg')}}"></td>
```
Comment (Kokosnootje) [inline resources/views/anime/DoNotTouchPlox/animepage.blade.php:59]: Touch plox

File: resources/views/anime/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@extends('layouts.app')
+
+
+@section('content')
+    <div class="row">
+        <div class="col-lg-12 margin-tb">
+            <div class="pull-left">
+                <h2>Products</h2>
```
Comment (okapifan) [inline resources/views/anime/index.blade.php]: Products?

**Review by okapifan — CHANGES_REQUESTED** [review]
Go fix

**Review by okapifan — APPROVED** [review]
You did well my bastard

**Review by Kokosnootje — APPROVED** [review]
Goed genoeg

**Pull request merged**